### PR TITLE
Pol 262 aws unused ip addresses

### PR DESCRIPTION
**Pull request by KVivek03**
### Description

AWS unused IP addresses policy

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: cost/aws/unused_ip_addresses/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+This Policy finds AWS unused IP addresses and deletes them after approval.
+
+## Functional Details
+
+This policy gets a list of EIP(Elastic IPs) which can be on the following platforms:
+
+- EC2-Classic- In this platform instances run in a single, flat network that is shared with other customers.
+- EC2- VPC- In EC2-VPC, instances run in a virtual private cloud (VPC) that is logically isolated to only one AWS account.
+
+Please note that all new AWS accounts are automatically on the EC2-VPC platform.
```
Comment (cdwilhelm) [inline cost/aws/unused_ip_addresses/README.md:14]: This sentence isn't necessary.

File: cost/aws/unused_ip_addresses/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# AWS Unused IP Addresses
+
+## What it does
+
+This Policy finds AWS unused IP addresses and deletes them after approval.
+
+## Functional Details
+
+This policy gets a list of EIP(Elastic IPs) which can be on the following platforms:
```
Comment (cdwilhelm) [inline cost/aws/unused_ip_addresses/README.md:9]: These details should describe what an unused IP address is.

Comment (KVivek03) [reply cost/aws/unused_ip_addresses/README.md]: made the change

File: cost/aws/unused_ip_addresses/README.md (hunk 1 above)
Comment (KVivek03) [reply cost/aws/unused_ip_addresses/README.md]: made the change

**Pull request merged**